From b9701b81ae21eb58136107080cdd6137bedcd3c9 Mon Sep 17 00:00:00 2001
From: Hu Wang <huw@codeaurora.org>
Date: Tue, 13 Apr 2021 17:57:04 +0800
Subject: [PATCH 1/3] wifi: Add vendor hostapd 1.2 hal to allowed_deps

This CL adds vendor hostapd 1.2 hal to allowed_deps to make
compiler happy.

CRs-Fixed: 2911466
Change-Id: Ic7c8200ab29a59d091773091c355e1f0e0d7c671
---
 build/allowed_deps.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build/allowed_deps.txt b/build/allowed_deps.txt
index 4567dee..e8abbba 100644
--- a/build/allowed_deps.txt
+++ b/build/allowed_deps.txt
@@ -723,6 +723,9 @@ TextClassifierNotificationLibNoManifest(minSdkVersion:30)
 TextClassifierServiceLibNoManifest(minSdkVersion:28)
 TextClassifierServiceLibNoManifest(minSdkVersion:30)
 tflite_support(minSdkVersion:30)
+vendor.qti.hardware.wifi.hostapd-V1.0-java(minSdkVersion:current)
+vendor.qti.hardware.wifi.hostapd-V1.1-java(minSdkVersion:current)
+vendor.qti.hardware.wifi.hostapd-V1.2-java(minSdkVersion:current)
 wifi-lite-protos(minSdkVersion:30)
 wifi-nano-protos(minSdkVersion:30)
 xz-java(minSdkVersion:current)
-- 
2.34.1

